pull_request_guidelines.md

```diff
@@ -0,0 +1,78 @@
+## Why do we do pull requests?
+
+* Catching and reducing technical debt
+* Catching mistakes
+* Improving development skills
+* Familiarity with the whole codebase
+* Maintaining consistency
+
+## Pull Request Software
+
+* Reviewboard vs. Github
+
+## Creating Pull Requests
+
+* Pull request granularity (how big should my pull requests be?)
+ * Try to keep them under 500 lines.
```

Comment:
Possibly address small PRs. Somewhat tied into the next point, "single responsibility principle", small PRs are ok as well, assuming they address a single issue. 

Starting out I know I felt that I sometimes needed to bundle small unrelated commits into a single PR, which in my opinion is not ideal.